Hardware/Hardware/CardReader/StripeCardReader/StripeCardReaderService.swift

```diff
@@ -62,14 +63,18 @@ extension StripeCardReaderService: CardReaderService {
 
     // MARK: - CardReaderService conformance. Commands
 
-    public func start(_ configProvider: CardReaderConfigProvider) {
+    public func start(_ configProvider: CardReaderConfigProvider) throws {
         setConfigProvider(configProvider)
 
         let config = DiscoveryConfiguration(
             discoveryMethod: .bluetoothProximity,
             simulated: false
         )
 
+        guard CBCentralManager.authorization != .denied else {
```

Comment:
There is also a `.restricted` case which is interesting. The docs say:

> In this state, the user can’t change the Bluetooth authorization status, possibly due to active restrictions such as parental controls

I'm not handling this currently since I imagine it's unlikely merchants will encounter that, and I understand that the person using the app might not be able to do anything about it.